packages/paste-website/src/components/tokens-list/index.tsx

```diff
@@ -42,10 +42,26 @@ export const TokensList: React.FC<TokensListProps> = (props) => {
     setFilterString(filter);
   };
 
+  const handleThemeChange = (e: React.ChangeEvent<HTMLSelectElement>): void => {
+    if (e.currentTarget.value === 'dark') setTokens(DarkModeTokens.tokens);
+    else if (e.currentTarget.value === 'default') setTokens(Tokens.tokens);
+    SimpleStorage.set('themeControl', e.currentTarget.value);
+  };
+
+  const handleFormatChange = (e: React.ChangeEvent<HTMLSelectElement>): void => {
+    if (e.currentTarget.value === 'javascript') setUseJavascriptNames(true);
+    else if (e.currentTarget.value === 'css') setUseJavascriptNames(false);
+    SimpleStorage.set('formatControl', e.currentTarget.value);
+  };
+
   if (tokens === null) {
     return <NoTokensFound onClearSearch={() => setFilterString('')} />;
   }
 
+  const backgroundColor =
+    (tokens['background-colors'].find((token) => token.name === 'color-background-body')?.value as string) ??
```

Comment:
praise: hadn't seen the `??` operator before, nice! Leaving myself this link of what it's called for future reference: [Nullish coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator) <-- also, is it me or does that name sound like an insult somehow? 😆

Reply:
hahaha I know, the name is funny. my FE mentor actually showed me this operator! definitely comes in handy